schemas.py

```diff
@@ -0,0 +1,36 @@
+from pydantic import BaseModel
+from datetime import date
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: str
+    publication_date: date
+
+
+class BookCreate(BookBase):
+    pass
+
+
+class Book(BookBase):
+    id: int
+
+    class Config:
+        orm_mode = True
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: str
+
+
+class AuthorCreate(AuthorBase):
+    book_id: int
+
+
+class Author(AuthorBase):
+    id: int
+    book: Book
```

Comment:
The `book` field in the `Author` schema should be a list of `Book` instances (i.e., `List[Book]`) to correctly represent the one-to-many relationship between authors and books.